src/Akeneo/Pim/Enrichment/Component/Category/Connector/ArrayConverter/FlatToStandard/Category.php

```diff
@@ -82,4 +90,38 @@ protected function convertField($convertedItem, $field, $data)
 
         return $convertedItem;
     }
+
+    /**
+     * @param array $item
+     * @return void
+     * @throws StructureArrayConversionException
+     */
+    protected function validate(array $item)
+    {
+        $requiredFields = ['code'];
+
+        $this->fieldChecker->checkFieldsPresence($item, $requiredFields);
+        $this->fieldChecker->checkFieldsFilling($item, $requiredFields);
+
+        $authorizedFields = array_merge($requiredFields, ['parent']);
+        // TODO pull-up 6.0/master
+        if (null !== $this->localeRepository) {
+            $locales = $this->localeRepository->findAll();
+            foreach ($locales as $locale) {
+                $authorizedFields[] = 'label-' . $locale->getCode();
+            }
+        }
```

Comment:
Like before do not check labels like before if no localeRepository
```suggestion
         // TODO pull-up 6.0/master remove this condition
        if (null === $this->localeRepository) {
            return;
        }
        
        $locales = $this->localeRepository->findAll();
        foreach ($locales as $locale) {
            $authorizedFields[] = 'label-' . $locale->getCode();
        }
```